app/controllers/attributes_controller.rb

```diff
@@ -0,0 +1,44 @@
+class AttributesController < ApplicationController
+  before_action :fetch_govuk_account_session
+
+  def show
+    attribute_names = params.fetch(:attributes)
+
+    access_token = @govuk_account_session[:access_token]
+    refresh_token = @govuk_account_session[:refresh_token]
+
+    values = attribute_names.each_with_object({}) do |name, hsh|
```

Comment:
nitpick, I see you assign it to the `values` variable here.
but not sure how I feel about `hsh` as shorthand. 

Could we shadow this as `values` here in the iterator, or at least use `hash`?

Total nitpick, take it or leave it.